pyproject.toml

```
@@ -18,9 +18,9 @@ repository = "https://github.com/Gandi/wagtail-oauth2"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-Django = "^3.2.4"
-wagtail = "^2.14.1"
-requests = "^2.26.0"
+Django = ">=3.2.0,<5"
```

Probably I can allow only Django 4.0 and not the next minor releases:

```suggestion
Django = ">=3.2.0,<4.1"
```

WDYT?

---

`<5` is ok to me